tests/PriorityMapTest.php

```diff
@@ -89,22 +91,33 @@ public function testGetKeys()
         $list->set('k4', 'v4', 1);
         $list->set('k5', 'v5', 4);
 
-        $this->assertEquals(
-            array('k1', 'k2', 'k3', 'k4', 'k5'),
-            $list->getKeys()
-        );
+        $result = $list->getKeys();
```

Comment:
my test checks that getKeys return keys in order of set, you now check that php has correct implementation of foreach:

```php
<?php
foreach($result as $key) {
            $this->assertContains($key, $result);
}
```
You iterate by result, get key from iterator, and check that this key present in result. hmmm.... i think something get wrong )))

Test must test something

Reply:
All right, I set the wrong parameters.
Thank you for yoir reply.